docs/content/en/docs/intro-klt.md

```diff
@@ -21,22 +37,32 @@ The Keptn Lifecycle Toolkit "wraps" a standard Kubernetes GitOps deployment
 with the capability to automatically handle issues
 before and after the actual deployment.
 
+KLT provides a number of features that can help you improve the observability and reliability of your applications. These features include:
+
+* **Pre-deployment checks:** KLT can run pre-deployment checks to ensure that your applications are ready to be deployed. 
+  These checks can include things like linting your code, running unit tests, and scanning for security vulnerabilities.
+* **Post-deployment monitoring:** KLT can monitor your applications after they have been deployed. This monitoring can 
+  include things like collecting metrics, tracing requests, and alerting on errors.
+* **Remediation:** If KLT detects that your application is not performing as expected, it can automatically remediate the 
```

Comment:
We should probably use "Day-2 Evaluation" rather than "Remediation".  The issue being that Keptn v1 has fairly sophisticated remediation facilities that are not provided in KLT, at least not yet.  Not very many people apparently use the full capabilities of remediation facilities but they are one reason someone might stay with Keptn v1 rather than going to KLT.  "Day 2 evaluation" or something applies to just running some evaluations on the production system a day or two (or more) after it has been released to make sure that everything is still healthy.